content_schemas/formats/popular_links.jsonnet

```diff
@@ -0,0 +1,40 @@
+{
+  document_type: "popular_links",
```

Comment:
I think there's a case to make that this should be named and implemented a bit more generically.

If we called it (say) `"link_collection"` then it could potentially be used by other content items that need to include a collection of links.

By calling it "popular links" we kind of restrict it to just this one use case.

In terms of implementation, I think we'd need to drop the `minItems: 6` / `maxItems: 6` constraint if we wanted it to be generic. I think that's fine - if we want exactly 6 items in the popular links document for the homepage, we can enforce that in Publisher. Just because we can validate it here, doesn't mean we should.

Reply:
Accepted